scripts/templates/print.hpp.mako

```diff
@@ -106,7 +108,9 @@ def findMemberType(_item):
     %endif
 </%def>
 
-namespace ${x}_params {
+namespace ${x}_print {
+## API functions declarations #################################################
+// RFC: Should is_handle stay in the common dir?
```

Comment:
as long as it's in a namespace, I don't see why not. Maybe we can move it to `details` (see below).